docs/reference/visibility-modifiers.md

```diff
@@ -13,18 +13,8 @@ There are four visibility modifiers in Kotlin:
 
 * `private` --- visible only in the declaring scope and its subscopes (inside the same module);
```

Comment:
I think this part about "its subscopes" no longer makes sense for package members, so probably we should just list the four modifiers here without explanation, given that they're thoroughly explained later